common-service-core/src/main/java/org/zowe/apiml/util/UrlUtils.java

```diff
@@ -101,5 +101,12 @@ public static String getIpAddressFromUrl(String urlString) throws MalformedURLEx
         String hostname = baseUrl.getHost();
         return UrlUtils.getHostIPAddress(hostname);
     }
-
+    public static boolean isValidURL(String url) {
```

Comment:
This class has the `UtilityClass` from lombok, so the `public` and `static` modifiers are not needed on the methods

Reply:
Actually `public` is not supplied by lombok so only `static` can be removed. Will be fixed in next commit.

Reply:
thanks